src/text-transform/transformers.ts

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+
+import {ConstructorBlock} from '../models/constructor';
+import {fullTransform} from './utils';
+
+import {Block} from '../models';
+
+import {Lang} from '../utils/configure';
+
+import {Transformer} from './common';
+
+import {config, BlocksConfig} from './config';
+
+export type ContentTransformerProps = {
+    content: {
+        blocks?: ConstructorBlock[];
+        footnotes?: string[];
+    };
+    options: {
+        lang: Lang;
+        customConfig?: {};
+    };
+};
+
+function addRandomOrder(block: ConstructorBlock) {
+    if (block) {
+        if ('randomOrder' in block && block.randomOrder && 'children' in block && block.children) {
+            block.children = _.shuffle(block.children as ConstructorBlock[]);
+        }
+    }
+}
+
+function transformBlocks(blocks: ConstructorBlock[], lang: Lang, customConfig = {}) {
+    const fullConfig = {...config, ...customConfig};
+
+    const transformedBlocks = _.cloneDeep(blocks);
```

Comment:
blocks are not transformed yet at this line

Reply:
fix name